platform/ui/src/components/ContextMenuMeasurements/ContextMenuMeasurements.jsx

```diff
@@ -26,31 +26,21 @@ const ContextMenuMeasurements = ({
         onSetLabel(item);
         onClose();
       },
-    },
-    {
-      label: 'Add Description',
-      actionType: 'setDescription',
-      action: item => {
-        onSetDescription(item);
-        onClose();
-      },
+      value: {}
     },
   ];
 
-  return <ContextMenu items={dropdownItems} />;
+  const menuItems = onGetMenuItems(defaultMenuItems);
+
+  return <ContextMenu items={menuItems} />;
 };
 
 ContextMenuMeasurements.propTypes = {
-  isTouchEvent: PropTypes.bool,
-  eventData: PropTypes.object.isRequired,
   onClose: PropTypes.func.isRequired,
-  onSetDescription: PropTypes.func.isRequired,
+  onSetDescription: PropTypes.func,
```

Comment:
Remove this and all mention of it if it's not being used.

Reply:
Removed.